src/script/calling/CallingRepository.ts

```diff
@@ -131,6 +130,7 @@ export class CallingRepository {
   private readonly acceptVersionWarning: (conversationId: QualifiedId) => void;
   private readonly callLog: string[];
   private readonly logger: Logger;
+  private enableBackgroundBlur = false;
```

Comment:
in a further feature, this will be moved to a proper setting in the `A/V` settings